custom-vu/src/main/kotlin/jces1209/vu/page/boards/view/cloud/CloudNextGenBoardPage.kt

```diff
@@ -53,6 +55,10 @@ class CloudNextGenBoardPage(
         return this;
     }
 
+    override fun getConfigureBoard(): ConfigureBoard {
```

Comment:
Stylistic preference:
```suggestion
    override fun configure(): BoardConfiguration {
```